openapi/src/openapi-src.json

```diff
@@ -631,18 +649,163 @@
                         "content": {
                             "application/vnd.api+json": {
                                 "schema": {
-                                    "$ref": "json-schemas/api/_questionnaires_{questionnaireId}_dataset/get/res/200.json"
-                                },
-                                "example": {
-                                    "data": [
+                                    "anyOf": [
                                         {
-                                            "type": "dataset",
-                                            "id": "0",
-                                            "attributes": {
-                                                "q-applicant-british-citizen-or-eu-national": true
-                                            }
+                                            "$ref": "json-schemas/api/_questionnaires_{questionnaireId}_dataset/get/res/200-v1.0.0.json"
+                                        },
+                                        {
+                                            "$ref": "json-schemas/api/_questionnaires_{questionnaireId}_dataset/get/res/200-v2.0.0.json"
                                         }
                                     ]
+                                },
+                                "examples": {
+                                    "v1.0.0": {
+                                        "summary": "Request Header \"Accept-Version: 1.0.0\"",
+                                        "value": {
+                                            "data": [
+                                                {
+                                                    "type": "dataset",
+                                                    "id": "0",
+                                                    "attributes": {
+                                                        "q-applicant-british-citizen-or-eu-national": true
+                                                    }
+                                                }
+                                            ]
+                                        }
+                                    },
+                                    "v2.0.0": {
+                                        "summary": "Request Header \"Accept-Version: 2.0.0\"",
+                                        "value": {
+                                            "data": [
+                                                {
+                                                    "type": "dataset",
+                                                    "id": "0",
+                                                    "attributes": {
+                                                        "values": [
```

Comment:
I'm wondering if we're not using the term "value" too often? Consumers of this object are going to have to navigate paths like: 

```dataset.values.forEach(value => {
    let answer = value.value;
})```

Not an huge issue but maybe something to consider?

Reply:
@BarryPiccinni the "[summary](https://github.com/CriminalInjuriesCompensationAuthority/data-capture-service/pull/224/files#diff-2836d18055d85134d1f93a05d9d4a74048ef60ddac08dee1ffb30d8b5c3c97f7R677)" and "[value](https://github.com/CriminalInjuriesCompensationAuthority/data-capture-service/pull/224/files#diff-2836d18055d85134d1f93a05d9d4a74048ef60ddac08dee1ffb30d8b5c3c97f7R678)" properties are part of the OpenAPI definition, not the actual response. Is that what you were referring to?